submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionDraftMongoPersistenceService.kt

```diff
@@ -57,9 +48,5 @@ class SubmissionDraftMongoService(
         key: String
     ) = draftDocDataRepository.setStatus(userEmail, key, PROCESSING)
 
-    private fun create(userEmail: String, key: String): DocSubmissionDraft {
-        val submission = toSubmissionMapper.toSimpleSubmission(queryService.getExtByAccNo(key))
-        val content = serializationService.serializeSubmission(submission, JsonPretty)
-        return draftDocDataRepository.createDraft(userEmail, key, content)
-    }
+    private fun DocSubmissionDraft.toSubmissionDraft() = SubmissionDraft(key, content)
```

Comment:
I do not see value in this function, is used ones and is at complex at just calling the constructor directly.